docs/sources/panels-visualizations/visualizations/alert-list/index.md

```diff
@@ -103,8 +107,8 @@ Filter alerts by the selected folder. Only alerts from dashboards in this folder
 
 Choose which alert states to display in this visualization.
 
-- Alerting / Firing
-- Pending
-- No Data
-- Normal
-- Error
+- **Alerting / Firing -** Displays alerts that are currently active and triggering an alert condition.
+- **Pending -** Shows alerts that are in a transitional state, waiting for conditions to be met before triggering.
+- **No Data -** Filters alerts where the data source is not returning any data, which could indicate an issue with data collection.
+- **Normal -** Displays alerts that are in a normal or resolved state, where no alert condition is currently met.
+- **Error -** Shows alerts where an error has occurred, typically related to an issue in the alerting process.
```

Comment:
Great addition here! We just want to keep the wording consistent; this helps readers skim rather than having to actually closely read every word on the page:
```suggestion
- **Alerting / Firing -** Shows alerts that are currently active and triggering an alert condition.
- **Pending -** Shows alerts that are in a transitional state, waiting for conditions to be met before triggering.
- **No Data -** Shows alerts where the data source is not returning any data, which could indicate an issue with data collection.
- **Normal -** Shows alerts that are in a normal or resolved state, where no alert condition is currently met.
- **Error -** Shows alerts where an error has occurred, typically related to an issue in the alerting process.
```